Modules/III-k6-Intermediate/10-Using-execution-context-variables.md

```diff
@@ -24,7 +24,7 @@ The **test** object allows you to abort the test execution with an optional erro
 
 The **VU** object assigns a unique number to every iteration and every VU in the load generator.
 
-To learn more about the `k6/execution` module, [click here.](https://k6.io/docs/javascript-api/k6-execution/).
+To learn more about the `k6/execution` module, [click here](https://k6.io/docs/javascript-api/k6-execution/).
```

Comment:
```suggestion
[Learn more about the `k6/execution` module](https://k6.io/docs/javascript-api/k6-execution/).
```
We always try to avoid "Click here", since it's not accessible. Screen readers often scan pages by URLs, so for them, browsing the page would give the auditory experience of "Click here", "Click here"...not very useful.

Beyond that, using descriptive link text is just more information rich and economical.